src/xdocs/google_style.xml

```diff
@@ -462,7 +462,7 @@
                             <td><a name="4.1.2"/><a href="#4.1.2"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s4.1.2-blocks-k-r-style">4.1.2 Nonempty blocks: K &amp; R style</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s4.1.2-blocks-k-r-style">4.1.2 Nonempty blocks: K &amp; R style</a>
                             </td>
```

Comment:
#3546 is approved and could be done separately from this PR.
Yes, please create an issue to make set of tokens be the same or at least analyse a diff